manage_breast_screening/clinics/presenters.py

```diff
@@ -54,8 +54,11 @@ def setting_name(self):
 
 class AppointmentListPresenter:
     def __init__(self, appointments, filter, counts_by_filter):
+        sorted_appointments = sorted(
```

Comment:
it would be more efficient to do this sorting in the ORM, with an `order_by` call. There's also [a way to set the default ordering](https://docs.djangoproject.com/en/5.2/ref/models/options/#ordering) - I think I might have added this for some things already but haven't done it consistently for all the models .